modules/rules/client/views/rules.client.view.html

```diff
@@ -44,14 +44,14 @@ <h4 class="media-heading">The topic.</h4>
                 </div>
               </li>
 
-              <li class="media">
+              <!-- <li class="media">
                 <div class="media-left">
                 </div>
                 <div class="media-body">
                   <h4 class="media-heading">The difficulty.</h4>
                   <b>This sets how difficult the prompts will be to draw and guess.</b> The random option will select prompts from all of the difficulties.
                 </div>
-              </li>
+              </li> -->
```

Comment:
Let's just delete the whole thing, rather than just commenting it out

Reply:
Fixed